aws/plugin.go

```diff
@@ -174,6 +174,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"aws_glue_crawler":                                             tableAwsGlueCrawler(ctx),
 			"aws_guardduty_detector":                                       tableAwsGuardDutyDetector(ctx),
 			"aws_guardduty_finding":                                        tableAwsGuardDutyFinding(ctx),
+			"aws_guardduty_filter":                                         tableAwsGuardDutyFilter(ctx),
```

Comment:
Please put this in alphabetical order (should be before `aws_guardduty_finding`)